data/transactions/logic/eval_test.go

```diff
@@ -1788,20 +1796,34 @@ int 1
 ==
 &&
 `
-	gtxnText := gtxnTextV2 + ` gtxn 0 ExtraProgramPages
+	gtxnTextV4 := gtxnTextV2 + ` gtxn 0 ExtraProgramPages
 int 0
 ==
 &&
 gtxn 1 ExtraProgramPages
 int 2
 ==
 &&
+`
+
+	gtxnText := gtxnTextV4 + `int 0
```

Comment:
Let's reduce further churn by naming the gtxnTextV5 now, rather than wait for a later one.